secret-handshake.json

```diff
@@ -0,0 +1,142 @@
+{
+    "#": [
+        "The exercise only lays out the \"commands\" method,", 
+        "but some language tracks have tested reversing the translation and",
+        "composing the two functions, as well.",
+        "Consider the sets not under \"commands\" to be optional."
+    ],
+    "commands": {
+        "description": ["Create a handshake for a number"],
+        "cases": [
+            {
+                "description": "shake for 1",
+                "input": 1,
+                "expected": ["wink"]
+            },
+            {
+                "description": "shake for 10",
+                "input": 2,
+                "expected": ["double blink"]
+            },
+            {
+                "description": "shake for 100",
+                "input": 4,
+                "expected": ["close your eyes"]
+            },
+            {
+                "description": "shake for 1000",
+                "input": 8,
+                "expected": ["jump"]
+            },
+            {
+                "description": "shake for 11",
+                "input": 3,
+                "expected": ["wink", "double blink"]
+            },
+            {
+                "description": "shake for 11 reversed",
+                "input": 19,
+                "expected": ["double blink", "wink"]
+            },
+            {
+                "description": "shake for all",
+                "input": 16,
+                "expected": ["wink", "double blink", "close your eyes", "jump"]
+            },
+            {
+                "description": "shake for all reversed",
+                "input": 31,
+                "expected": ["jump", "close your eyes", "double blink", "wink"]
+            },
+            {
+                "description": "shake for binary",
+                "input": "101",
```

Comment:
it may be difficult for statically-typed languages to handle the fact that the value for the key "input" can be either a string or an integer. I believe @kytrinyx has already asked about whether we should double up on the binary conversion though.